lib/src/signed_video_internal.h

```diff
@@ -53,6 +53,7 @@ typedef struct _h26x_nalu_list_t h26x_nalu_list_t;
 
 #define DEFAULT_MAX_GOP_LENGTH 300
 #define RECURRENCE_MIN 1
```

Comment:
I think we should rename "RECURRENCE_MIN" to "RECURRENCE_ALWAYS" or something to indicate that it means it is always present.